scopes/defender/linter/linter.service.tsx

```diff
@@ -29,6 +30,7 @@ export class LinterService implements EnvService<LintResults> {
     const linterContext: LinterContext = Object.assign(
       {},
       {
+        rootDir: this.workspace.path,
```

Comment:
Does it makes sense to pass the whole workspace aspect to the linter service if only the path field is needed? Maybe just pass the path to the constructor instead?

Reply:
agree, done